app/analysis/index/controller.js

```diff
@@ -193,6 +193,8 @@ const AnalysisController = Controller.extend(AnalysisQueryParams.Mixin, {
 
   _scrollOffset: 0,
 
+  dataIntegrity: true,
```

Comment:
As discussed: This deserves a comment.

Reply:
Added an comment, check if it makes it more understandable

Reply:
:+1: